tools/cli/commands.js

```diff
@@ -925,7 +925,8 @@ on an OS X system.");
       //     packages with binary npm dependencies
       serverArch: bundleArch,
       buildMode: options.debug ? 'development' : 'production',
-    }
+    },
+    providePackageJSONForUnavailableBinaryDeps: !!process.env.METEOR_BINARY_DEP_WORKAROUND,
```

Comment:
eh, env vars are a little easier to set once in your shell permanently instead of having to type it every time you deploy?

I kinda think we should (later?) also activate it if DEPLOY_HOSTNAME=galaxy.meteor.com but @n1mmy suggested not yet.